src/main/java/com/visenze/visearch/PagedSearchResult.java

```diff
@@ -25,6 +25,10 @@
 
     private List<GroupSearchResult> groupSearchResults;
 
+    private String recommendationStrategy;
```

Comment:
i think the API specs has changed, it should be `recommendation_algorithm` now based on Mingyuan's doc.